solutions/kubernetes/functions.sh

```diff
@@ -218,115 +218,4 @@ function ssh_all_nodes() {
 
 function ssh_primary_node() {
     ssh -o 'StrictHostKeyChecking=no' "$PRIMARY_NODE" $*
-}
-
-function setup_awscli() {
```

Comment:
IO Sanity has same functions implemented, which is used for both ceph/cortx deployment. Are these same functions used by others from functions.sh instead of io-sanity.sh script?
Shouldn't there be one script for aws-setup etc (io-sanity.sh) instead of two different script?

Reply:
Ok. but then this change is not related to CORTX-33572: RADOS CI/CD Notification Modifications. Let's take this in separate PR. We need to validate all CORTX deployment scripts work fine for this change.

Reply:
Ok reverted functions.sh change.